tensorflow/python/platform/sysconfig.py

```diff
@@ -51,5 +53,32 @@ def get_lib():
   import tensorflow as tf
   return _os_path.join(_os_path.dirname(tf.__file__))
 
+
+def get_compile_flags():
+  """Get the compilation flags for custom operators.
+
+  Returns:
+    The compilation flags.
+  """
+  import tensorflow as tf
```

Comment:
For this, I think you should import versions.py direction from the top.

Reply:
Can you clarify what do you mean?

Reply:
What I meant to say is that you don't have to import tf here. I know it's done above but it's a special case. You can say `from tensorflow.python.framework.version import ...`